internal/server/meta_init.go

```diff
@@ -326,7 +327,7 @@ func sinkConnectionHandler(w http.ResponseWriter, r *http.Request) {
 		handleError(w, err, "", logger)
 		return
 	}
-
+	config = replacePasswdForSinkConfig(sinkNm, config)
```

Comment:
SinkConnectionHandler, SourceConnectionHandler, AND all PUT action for sink/source/connection are needing this conversion.